src/oc/storage/api/entries.clj

```diff
@@ -60,6 +60,26 @@
         [false, {:reason (.getMessage e)}])) ; Not a valid new entry
     [false, {:reason "Invalid board."}])) ; couldn't find the specified board
 
+(defn- clean-polls-for-patch
+  "Given the new"
+  [existing-entry entry]
+  (update entry :polls
+   (fn [polls]
+    (let [updated-polls (mapv (fn [[k poll]]
```

Comment:
any reason for the use of `mapv` in here rather than `map`?

Reply:
Also I'm thinking this is probably better as a named fn rather than an anonymous fn.

Reply:
> any reason for the use of `mapv` in here rather than `map`?

Just a test!